flume-ng-sinks/flume-hdfs-sink/pom.xml

```diff
@@ -191,6 +191,37 @@ limitations under the License.
 
       </dependencies>
     </profile>
+
+    <profile>
+      <id>hbase-1.0.0</id>
+      <activation>
+        <property>
+          <name>hadoop.profile</name>
```

Comment:
The "hadoop.profile" is the option which is being used in flume to set Hadoop related profile while compile/test/package/etc. phases. The community earlier added "hbase-98" profile which uses dependencies of hadoop-2 but with changes in HBase dependencies that were made in HBase pom (splitting hbase.jar to hbase-client.jar and hbase-server.jar, etc.). So for hbase-1.0.0, I also followed the same approach and provided profile name as "hbase-1.0.0".